app/views/sciruby_github/repos.erb

```diff
@@ -2,18 +2,38 @@
 <div class="span12 pagination-centered" style="text-align: center;">
     <h3> SciRuby github data analysis (Using daru-view/googlecharts)</h3>
     <b>Data is taken from url : https://api.github.com/orgs/Sciruby/repos </b>
-    <%= render 'export' %>
+    <%=
+      daru_download formats: %w[csv xls json], separator: '|'
+    %>
     <p style="left: 50%">
       <b>SciRuby Datatable : </b>
-      <%=raw @df_repo_table.div %>
+      <%=
+        daru_table @df_repos,
+                   adapter: :googlecharts,
+                   pageSize: 10,
+                   height: 300,
+                   width: 400
```

Comment:
This (and `daru_chart` below) seems for me:
a) pretty important
b) pretty easy to implement (just what you already have + small helper method).

Ideas here:
1. when Rails guy sees something like `raw something`, it feels like a dirty hack.
2. when tutorial says that to show something in view, you should prepare it in controller, it is "wrong" concern separation
3. when "view-related" options are stored in controller (width, height, and so on) -- it also feels wrong (imagine there are some header image above, which should be exactly the width of the table; now, you replace image with another one and think "Oh, I need to make table 50px wider... I'll just fix my view. What??? Where is this width param?.."). 
4. this way (helper method in view) feels most "natural": "OK, I've added some _view_ library, now I have some helper in _view_ and here is its params".